pkgs/os-specific/linux/unstick/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, meson, ninja, pkgconfig, libseccomp }:
+
+stdenv.mkDerivation rec {
+  name = "unstick";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "kwohlfahrt";
+    repo = name;
+    rev = "5691ace9d210f506571bb1312ffca79216e4dfff";
+    sha256 = "10vf6v6252b78jyzwx7ihbf60vz89l5wxczjmyhz7jd0kir1bqyn";
+  };
+
+  sourceRoot = "source/src";
+
+  nativeBuildInputs = [ meson ninja pkgconfig ];
+  buildInputs = [ libseccomp ];
+
+  meta = {
+    homepage = https://github.com/kwohlfahrt/unstick.git;
```

Comment:
It's wonderful you've created this software especially for Nix. Your contribution is appreciated! Please quote the URL and remove the `.git` suffix.

Reply:
The `.git` is definitely out of place, fixed.

The majority of URLs on the `homepage` attribute across nixpkgs are unquoted (10k unquoted, 2k quoted according to `git grep`), is there any style guide for things like this?

Reply:
Yes. See https://github.com/NixOS/rfcs/pull/45

Reply:
Cool, fixed.